nixos/modules/services/continuous-integration/jenkins/default.nix

```diff
@@ -254,6 +254,33 @@ in
         StateDirectory = lib.mkIf (lib.hasPrefix "/var/lib/jenkins" cfg.home) "jenkins";
         # For (possible) socket use
         RuntimeDirectory = "jenkins";
+        AmbientCapabilities = "";
+        CapabilityBoundingSet = "";
+        LockPersonality = true;
+        # MemoryDenyWriteExecute = false;   Breaks execution;
+        NoNewPrivileges = true;
+        PrivateDevices = true;
+        PrivateMounts = true;
+        PrivateTmp = true;
+        ProtectClock = true;
+        ProtectControlGroups = true;
+        ProtectHome = true;
```

Comment:
@felixsinger This one breaks service start with non-default home `services.jenkins.home = "/home/jenkins"`:
```
rm: cannot remove '/home/jenkins/war': Permission denied
```

Reply:
Thank you for bringing it up. I will take care of it.

Reply:
I don't see a reason why Jenkins should be able to access `/home` directories. That is considered bad practice and I think enabling this is good as is. If you really need this disabled, then I suggest to keep overriding it.